### PR TITLE
dotnet6-sdk@6.0.423: Fix hash

### LINKED ISSUE #166: dotnet6-sdk@6.0.423: hash check failed
Checking hash of dotnet-sdk-6.0.423-win-x64.zip ... ERROR Hash check failed!
App:         business/dotnet6-sdk
URL:         https://dotnetcli.blob.core.windows.net/dotnet/Sdk/6.0.423/dotnet-sdk-6.0.423-win-x64.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    2fae66117bca22d76afde89e91bf8a40b4602254db4f831b0647e69e3ced23f972f335521847955069e6b8a74e7320ffe2af1f2bfc48842216451216845c3301
Actual:      34d4000c23822c92f8d6c5d72d65036544e7c3bfb958d3fea89da34e185df80c22ca0cbcd789725c90e935c71dcef2a361ca909bbe49c61fffd6984f211a22f4

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #166

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### dotnet6-sdk

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
